wiki/Difficulties/en.md

```diff
@@ -13,10 +15,10 @@ Star Rating and Web Icons
 
 This is the range that decides which Web Icons are used depending on the star rating of any given map. Following the icon range is a good estimate to achieve a balanced difficulty spread. There are some guidelines you can follow to improve the difficulty spread of your mapset.
 
-[Standard](Standard) difficulty spreads
+[Standard](/wiki/Game_Modes/osu!) difficulty spreads
```

Comment:
According to the ASG, section headings are not to have links.
Change to `osu! difficulty spreads` (or `osu!standard difficulty spreads`)